docs/ntppool/zh_CN/join.html

```diff
@@ -0,0 +1,75 @@
+[% page.title = '加入NTP服务器群！'  %]
+
+<div class="block">
+	<h3><a name="join">我该如何加入pool.ntp.org？</a></h3>
+	<p>
+	首先，我们欢迎你的加入。随着时间服务器群的用量的增加，为了减轻各个服务器的负载，
+	我们能做到的便是增加群内服务器的数量。
+	</p>
+
+	<p>
+	你的电脑<b>必须拥有一个静态的IP地址</b>，并且要有持续不断的，稳定的网络连接。
+	你的IP地址不变或几乎不变对我们来说很重要（比方说，一年换一次IP）。我们对你的网速要求很低。
+	专业一点说，每个用户每几分钟或十几分钟只会与你交换数个UDP数据包。
+	</p>
+
+	<p>
+	目前多数的服务器每秒会收到大概5-15个NTP数据包，高峰期则会增长到每秒大概60-120个NTP数据包。
+	折合成网速大概是1-2KB/s （10-15Kbit/s），高峰期大概8-20KB/s （50-120Kbit/s）。
+	鉴于我们的服务器数量一直在稳步增长，每个服务器的负载理应不会大幅增长。
+	换句话说，你大概有384-512Kbit的上下行网速就够用了。
+	</p>
+	<p>
+	点击数字查看随机一个服务器的负载图表：
+	[% i = 0; FOR server = combust.servers_with_urls; %]
+	   <a href="/scores?ip=[% server.ip %]">[% i=i+1; i %]</a>
+	[% END %]
+	</p>
+
+	<p>
+	欲加入时间服务器群的用户不应将<tt>pool.ntp.org</tt>设置为获取时间的上游服务器。
+	相反，你应该自行选择并设置一些
+	<a href="http://support.ntp.org/bin/view/Servers/StratumTwoTimeServers">优质的服务器</a>
+	作为时间来源。尽管列表中的一部分服务器<b>可能</b>依然在服务器群中，
+	但直接选择一个固定的服务器要比由系统随机分配稳定的多。这将有助于我们将准确的时间提供给大众。
+
+	请注意：由于我们服务器群的主要目的是分散负载，所以我们并不会在意你的时间服务器所处的层级。
+	</p>
+
+	<p>
+	我们为你准备了<a href="/join/configuration.html">加入NTP服务器群的推荐系统设定</a>。
+	</p>
+	
+	<p>最后，我一定要重申一下：加入NTP服务器群一定要有<b>长期规划</b>。
+	如果你想退出，我们会很乐意协助你。然而基于NTP客户端的运行模式，<b>将你的服务器上的用户全部转移走的过程可能要持续数周，数月，甚至数年</b>。
+	</p>
+
+	<p>
+	如果你觉得万事俱备了，那么请登陆<a href="/manage">服务器管理</a>系统后申请加入时间服务器群。
+	如果你在使用服务器管理系统时遇到了任何问题，请给我发电子邮件（英文）：<a
+	href="mailto:ask@develooper.com">ask@develooper.com</a>.
+	</p>
+	<p>
+	我希望你能够同时订阅<a href="https://news.ntppool.org/atom.xml">网站日志源</a>
+	和<a href="https://lists.ntp.org/listinfo/pool">服务器群邮件列表</a>.
+	</p>
+	<p>
+	（可选）如果你能把你服务器80端口的网页请求全部重定向至官方主页
+	<tt>http://www.pool.ntp.org</tt>，我们将对其感激不尽。
+	如果你的服务器上运行着阿帕奇（Apache），你可以通过以下方式完成重定向:
+	</p>
+
+        [% PROCESS tpl/join/virtual_host_example.html %]
+
+	<p>
+	如果你有能力将网页请求重定向，请谨记：我们的官网前面是一定要加上“www”的。
+	如果忘掉了“www”而直接访问<tt>pool.ntp.org</tt>，有可能会打开一个随机的页面哦。
```

Comment:
No, my friend, no. This is part not about the `www` in the redirection target, it's about why they want us to set up a redirection.